easyrecyclerview/src/main/java/com/jude/easyrecyclerview/EasyRecyclerView.java

```diff
@@ -380,11 +386,44 @@ public void setRefreshingColor(int... col) {
      *
      * @param listener
      */
+
+    /**
+     * Set scroll listener to the recycler
+     *
+     * @deprecated Use {@link #addOnScrollListener(RecyclerView.OnScrollListener)} instead.
+     */
+    @Deprecated
```

Comment:
In these types of cases, we shouldn't remove functionality. It might break other people's projects if they are directly accessing the `mExternalOnScrollListener`.